LICENSE

```diff
@@ -1,6 +1,6 @@
 MIT License
 
-Copyright (c) 2021 DataJoint
+Copyright (c) 2020 Thinh Nguyen
```

Comment:
work for hire, should be vathes

Reply:
oops, artifact from copying over from my original repo

Reply:
"DataJoint Contributors"